sci-biology/salmon/salmon-0.5.0.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
cmake-utils does not yet support EAPI=6, that's the reason why repoman fails. Patches are on the way.